Odata-docs/webapi/pagesize-top-maxtop.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "PageSize, Top and MaxTop"
+description: "This tutorial describes how to configure PageSize, Top and MaxTop."
+
+author: KenitoInc
+ms.author: kemunga
+
+ms.date: 3/1/2022
+---
+# PageSize, Top and MaxTop
+**Applies To**:[!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v7.md)][!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v6.md)]
+
+In OData WebApi we support both client-driven and server-driven paginations.
+
+### Client-driven pagination
+In client-driven paging, we request the server to return the specified number of results. There is no `nextLink` that is returned.
+
+The OData WebApi deals with client-driven paging using `$skip` and `$top` query options.
+
+The `$top` query option requests the number of items in the queried collection to be included in the result.
+
+The `$skip` query option requests the number of items in the queried collection that are to be skipped and not included in the result.
+
+The `MaxTop()` sets the maximum value that can be set for the `$top` query parameter. It does not directly limit the number of items returned per request.
+
+In `Microsoft.AspNetCore.OData v7.x`, we configure `MaxTop()` as follows:
+
+```csharp
+app.UseMvc(routeBuilder =>
+{
+    routeBuilder.Select().Expand().Filter().OrderBy().MaxTop(100).Count();
```

Comment:
```suggestion
    routeBuilder.Select().Expand().Filter().OrderBy().MaxTop(20).Count();
```
For those skimming the docs the error would look out of place. You could change the integer to indicate the maxtop of 100 was exceeeded.